onnxruntime/contrib_ops/cuda/math/gemm_float8.cu

```diff
@@ -0,0 +1,472 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "contrib_ops/cuda/math/gemm_float8.h"
+#include "core/providers/cuda/cu_inc/common.cuh"
+#include "core/providers/cuda/shared_inc/cuda_utils.h"
+#include <algorithm>
+#include <utility>
+#include <cuda_runtime.h>
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+static const char* cublasGetErrorEnum(cublasStatus_t error) {
```

Comment:
There is already a definition here: https://github.com/microsoft/onnxruntime/blob/83547d30672ca1fa11327420037a1be2a6448950/onnxruntime/core/providers/cuda/cuda_call.cc#L34. 

Could you please leverage it by moving it to a common place?